diff --git a/E6553_ramdisk/init.sony-platform.rc b/E6553_ramdisk/init.sony-platform.rc
--- a/init.sony-platform.rc
+++ b/init.sony-platform.rc
@@ -15,8 +15,8 @@ on early-init
 on init
     # SONY: Start the TrimArea Daemon. It must be started before fota-ua
     wait /dev/block/mmcblk0p1
-    chown tad tad /dev/block/mmcblk0p1
-    chmod 0770 /dev/block/mmcblk0p1
+    chown root root /dev/block/mmcblk0p1
+    chmod 0777 /dev/block/mmcblk0p1
     class_start trimarea
     exec u:r:tad:s0 system -- /sbin/wait4tad_static
 
@@ -305,8 +305,8 @@ on property:ro.semc.enable.fast_dormancy=false
 # SONY: TrimArea Daemon
 # Last 2 args: start block(blk size 128k), number of blocks(partitionsize(kb)/128(kb))
 service tad_static /sbin/tad_static /dev/block/bootdevice/by-name/TA 0,16
-    user tad
-    group tad root
+    user root
+    group root root
     socket tad stream 0660 system trimarea
     class trimarea
 # tad_static is in rootfs, normal file_context does not work
